openquake/calculators/postproc/compute_rtgm.py

```diff
@@ -326,6 +340,284 @@ def get_asce41(dstore, mce, facts):
+
+        # save single imt plot
+        bio1 = io.BytesIO()
+        fig1.savefig(bio1, format='png', bbox_inches='tight')
+        # keep these in webui until we finish checks and have a command line
+        # exporter, then we can change the name to _{imt} and they will not
+        # appear in the webui
+        dstore[f'png/disagg_by_src-{imt}.png'] = Image.open(bio1)
+
+    # save triple plot
+    bio = io.BytesIO()
+    fig.savefig(bio, format='png', bbox_inches='tight')
+    logging.info('Storing png/disagg_by_src.png')
+    dstore['png/disagg_by_src-All-IMTs.png'] = Image.open(bio)
+
+
+def plot_governing_mce(dstore, imtls):
+    imt_list = []
+    imt_list = []
+    imls = []
+    for imt, iml in imtls.items():
+        imls.append([im for im in iml])
+        imt_list.append(imt)
+    js = dstore['asce7'][()].decode('utf8')
+    dic = json.loads(js)
+    MCEr_det = [dic['PGA_84th'], dic['SS_84th'], dic['S1_84th']]
+    MCEr = [dic['PGA'], dic['SS'], dic['S1']]
+    T = [from_string(imt).period for imt in imt_list]
+
+    limit_det = [0.5, 1.5, 0.6]
+    # presenting as maximum component -> do not need conversion facts
+    RTGM = dstore.read_df('rtgm')['ProbMCE']
+    plt.figure(figsize=(8, 6))
+    plt.rcParams.update({'font.size': 15})
+    plt.plot(T, limit_det, 'kx', markersize=15, label='DLL', linewidth=1)
+    plt.plot(T[0], RTGM[0], 'bX', markersize=12, label='$PGA_{GM}$',
+             linewidth=3)
+    plt.plot(T[1:], RTGM[1:], 'bs', markersize=12,
+             label='$S_{S,RT}$ and $S_{1,RT}$', linewidth=3)
+    plt.plot(T[0], MCEr_det[0], 'c^', markersize=10, label='$PGA_{84th}$',
+             linewidth=3)
+    plt.plot(T[1:], MCEr_det[1:], 'cd', markersize=10,
+             label='$S_{S,84th}$ and $S_{1,84th}$', linewidth=3)
+    plt.scatter(T[0], MCEr[0], s=200, label='Governing $MCE_G$',
+                linewidth=2, facecolors='none', edgecolors='r')
+    plt.scatter(T[1:], MCEr[1:], s=200, marker='s',
+                label='Governing $MCE_R$', linewidth=2,
+                facecolors='none', edgecolors='r')
+    plt.grid('both')
+    plt.ylabel('Spectral Acceleration (g)', fontsize=20)
+    plt.xlabel('Period (s)', fontsize=20)
+    plt.legend(loc="upper right", fontsize='13')
+    plt.ylim([0, np.max([RTGM, MCEr_det, MCEr, limit_det]) + 0.2])
+    plt.xlim([-0.02, 1.2])
+    bio = io.BytesIO()
+    plt.savefig(bio, format='png', bbox_inches='tight')
+    plt.clf()
+    return Image.open(bio)
+
+
+def plot_curves(dstore):
+    dinfo = get_info(dstore)
+    # site is always 0 for a single-site calculation
+    # get imls and imts, make arrays
+    imtls = dinfo['imtls']
+    # separate imts and imls
+    AFE, afe_target, imls = [], [], []
+    # NOTE: for meanHCs_afe_RTGM and disaggr_by_src we want to display these
+    # three imts, that are mandatory in this context. For the plot of governing
+    # MCE we read imts from the imtls
+    imt_list = ['PGA', 'SA(0.2)', 'SA(1.0)']
```

Comment:
We already have the constant `imts`

Reply:
replaced `imt_list` with `imts` except for in `plot_governing_mce`